cristin-import/src/test/java/no/unit/nva/cristin/CristinDataGenerator.java

```diff
@@ -437,7 +444,7 @@ private static CristinJournalPublication randomJournalPublictaion() {
                 .withPagesBegin("1")
                 .withPagesEnd(String.valueOf(smallRandomNumber()))
                 .withVolume(String.valueOf(smallRandomNumber()))
-                .withDoi(String.valueOf(smallRandomNumber()))
+                .withDoi(randomDoiString())
```

Comment:
Use a random Doi String satisfies the DoiValidator criteria for offline validation. Online validation is disabled during testing